packages/communication-react/review/communication-react.api.md

```diff
@@ -582,7 +572,6 @@ export type ChatCompositeIcons = Partial<Pick<CompositeIcons, 'MessageDelivered'
 // @public
 export type ChatCompositeOptions = {
     errorBar?: boolean;
-    participantPane?: boolean;
```

Comment:
@JamesBurnside looks like api-extract isn't respective the `@beta` tag for this field of an `@public` interface.

The documentation on this is a bit unclear: 
![image](https://user-images.githubusercontent.com/82062616/142484067-27140a7d-3b6f-484d-9fe5-d26c657e4a47.png)

from https://api-extractor.com/pages/tsdoc/doc_comment_syntax/#release-tags

My reading of that was that we *should* be able to restrict the fields in a type to  more restrictive release tag, but that doesn't seem to be working.

We need to fix this regardless of my PoC PR here. Currently `ChatCompositeOptions.participantPane` is part of public API (though with the default value of false that we wanted)

@PorterNan our resident API change czar at the moment.

Reply:
Not sure I follow, api-trimming will trim out the participant pane if its marked @beta:
![image](https://user-images.githubusercontent.com/2684369/142484507-ab01c862-78c7-486b-a3d9-d072b28eb58a.png)

There's a difference between the api.md file and the .d.ts file though (api.md isn't trimmed - but is only used for us reviewing things)

Reply:
(though the fact it doesn't show a `//beta` beside it in the api.md file looks definitely like a bug on them)

Reply:
Ah, OK then.

See my comment on your beta trimming PR for how we should generate separate APIview files for the trimmed types as well.